pages/home.jsx

```diff
@@ -275,6 +275,12 @@ var HomePage = React.createClass({
     router: React.PropTypes.func.isRequired
   },
   componentDidMount: function() {
+    // auto pops up the Pledge modal if the user is visiting 
+    // the homepage for the first time
+    if (localStorage.getItem("autoOpenPledgeModal") !== "false") {
```

Comment:
I think I'm gonna rename the var to "disableAutoOpenPledgeModal" and do
```js
if (!localStorage.getItem("disableAutoOpenPledgeModal")) {
  this.handlePledgeBtnClick();
  localStorage.setItem("disableAutoOpenPledgeModal", "true");
}
```

this way the logic looks more intuitive.  agree? @Pomax